tools/scylla-sstable.cc

```diff
@@ -192,7 +193,9 @@ struct schema_with_source {
     sstring obtained_from;
 };
 
-std::optional<schema_with_source> try_load_schema_from_user_provided_source(const bpo::variables_map& app_config, db::config& cfg) {
+std::optional<schema_with_source> try_load_schema_from_user_provided_source(const bpo::variables_map& app_config,
+                                                                            db::config& cfg,
+                                                                            const ::utils::directories& dirs) {
```

Comment:
nit: this kind of line-breaking is discouraged, as soon as the name of the method changes, the lines become unaligned.

Just align all wrapped lines, with two extra ident, compared to the original line.

Reply:
Sure, fixed. Thanks :+1: